### DISCUSSION TITLE
Add "Daily Hijri Date" sender and upcoming events.

### DISCUSSION THREAD

**Discussion by Nate-Champion** [fivekage/muslimbot#62] [Ideas]
I liked your bot. It will be nice to add a new feature. The bot should send daily Hijri Date with upcoming events. 
I have attached example screenshot. 

![Screenshot_20240803-185639](https://github.com/user-attachments/assets/b8f56ca0-c268-4921-95a5-7190c2364180)

**Comment by samymokhtari** [comment]
Hello my friend, thank you for your suggestion! Yea it will be a great idea what do you suggest for the command ?
```
/today
/news
.. ?
🙂
